site/docs/getting-started/designing/index.mdx

```diff
@@ -132,6 +132,21 @@ Salt offers six libraries in Figma. Install the library that best suits your nee
 
 If you are migrating from other libraries, including Salt (Next), Salt Light, Dark libraries, and UITK, please refer to our [migration guide](https://go/salt-ds-internal/migration-strategies/salt-ds-figma-libraries/index) (JPMC employees only).
 
+## What are Salt Figma Template components?
```

Comment:
Are they going to be known as Figma Template components or just Figma Templates?

Reply:
I'm not sure if the underlying mechanism is important to convey in the name?

The paragraph already says it's a figma component